Automattic/wp-calypso
# Plugins: Fix the spacing of the upgrade banner

## Proposed Changes
Change the styling of the upgrade banner container to add top padding

## Testing Instructions



1. In Calypso go to Plugins
2. Inspect the upgrade banner and make sure it the necessary top padding

## After
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/20927667/186226917-d577b845-71fc-43d9-bc15-b0410c86750b.png">


Fixes #66484
